Makefile

```diff
@@ -17,16 +18,12 @@ COMPOSE_IGNORE_ORPHANS=1
 COMPOSE_PROJECT_NAME ?= grapl
 export
 
-export EVERY_LAMBDA_COMPOSE_FILE=--file docker-compose.lambda-zips.js.yml \
-	--file docker-compose.lambda-zips.rust.yml
-
```

Comment:
This was used for the heavy-handed make down target, but it's no longer needed because the containers now clean up after themselves. Example below in line 161: https://github.com/grapl-security/grapl/pull/1066/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R161